cosmoz-omnitable-styles.js

```diff
@@ -132,7 +132,28 @@ export default css`
 			--paper-font-caption_-_line-height: 18px;
 		}
 	}
+	cosmoz-omnitable-amount-range-input,
+	cosmoz-omnitable-number-range-input,
+	cosmoz-omnitable-date-range-input,
+	cosmoz-omnitable-time-range-input,
+	cosmoz-omnitable-datetime-range-input,
+	paper-input {
+		--paper-font-subhead_-_font-family: var(
+			--cosmoz-header-font-family,
+			'Inter',
+			sans-serif
+		);
+		text-transform: var(--cosmoz-header-text-transform, uppercase);
```

Comment:
You should include the component name in the css var, so it is clear when someone is reading the CSS the purpose of a certain variable.

```suggestion
		text-transform: var(--cosmoz-omnitable-header-text-transform, uppercase);
```